update quickstart & wallet (#90)
hyli-org/devhub-hyli: docs/concepts/apps.md

Apps on Hyli rely on a pairing of a zero-knowledge proof scheme (verifier) and a program identifier. Together, these components validate contract proofs.

> See the verifier scheme for each supported proving scheme in t[his reference page](../reference/supported-proving-schemes.md).

Reviewer: ```suggestion
See the verifier scheme for each supported proving scheme in [this reference page](../reference/supported-proving-schemes.md).
```
```suggestion
See the verifier scheme for each supported proving scheme in t[his reference page](../reference/supported-proving-schemes.md).
```
